examples/library/bayesian_active_learning/bayesian_active_learning.py

```diff
@@ -884,29 +1028,58 @@ def __call__(self, filepaths, input_style='json'):
     comm.Barrier()
 
 if rank==0:
-    with open('structures_chosen.dat', 'w') as f:
+    with open(AL_settings.output_directory+'structures_chosen.dat', 'w') as f:
         for i in range(len(structures_chosen_list)):
             f.write(str(i)+  ' : ' + ', '.join('/'.join(groupconfig) for groupconfig in structures_chosen_list[i]) + '\n')
 
+if parallel:
+    comm.Barrier()
+if rank==0:    
+    snap.write_output()
+    copy2(snap.config.sections["OUTFILE"].potential_name + '.snapcoeff', AL_settings.output_directory+snap.config.sections["OUTFILE"].potential_name + '.snapcoeff')
```

Comment:
This line causes an error on my end because `AL_settings.output_directory` is a path and `snap.config.sections["OUTFILE"].potential_name` is another path, but the combination of these paths doesn't exist. Looking for a workaround.

Reply:
@ldwillia if I comment out these `copy2` lines, it's all good. Are they necessary?